<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>315</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>2040</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>load your image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2_pic">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Dimensions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dimension_values">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>W:0 H:0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_edge_pic">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>360</width>
      <height>360</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge detection</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_edgeMethod">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sobel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Canny</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="slider_lowThreshold">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_highThreshold">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider_thresolding">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>720</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6_thresold">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6_thresold_high">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>high: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6_thresold_low">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>low: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Method</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>493</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
